lib/steep/drivers/watch.rb

```diff
@@ -126,6 +126,10 @@ def run()
           client_writer.write({ method: :shutdown, id: 10000 })
           client_writer.write({ method: :exit })
           client_writer.io.close()
+          Thread.start do
+            sleep 1
+            master.interrupt if main_thread.alive?
```

Comment:
Currently, if the running process was interrupted, `shutdown` and `exit` messages are sent.

I added to sleep for a while then interrupt the master worker because sending messages does not mean worker processes stops immediately.   `master.interrupt` yields all child worker processes are stopped soon.